build.sh

```diff
@@ -150,7 +152,7 @@ if buildAll || hasArg nvstrings; then
         python setup.py build_ext
         python setup.py install --single-version-externally-managed --record=record.txt
     else
-        python setup.py build_ext --library-dir=${LIBNVSTRINGS_BUILD_DIR}
+        python setup.py build_ext --build-lib=build/ --library-dir=${LIBNVSTRINGS_BUILD_DIR}
```

Comment:
Ideally this would just be `--inplace` instead of `--build-lib=build/` but I know there's CMake issues for the NVStrings python extensions. Given it's going away I don't think it makes sense to try to resolve that, but would it be possible to change `build/` to the current working directory to mirror `--inplace`?

Reply:
I tried `--inplace` and found that it expects  `pyniNVText.cpython-37m-x86_64-linux-gnu.so`. But Cmake extension creates file as `pyniNVText.so`. So, copy failed. 

> Given it's going away I don't think it makes sense to try to resolve that,

Is there a PR for this?
https://github.com/karthikeyann/cudf/blob/06a5e91e0c9a59b30068848a3cd953fecad02228/python/nvstrings/setup.py#L34

Removing this line should make `--inplace` work.

Reply:
Without that line I don't think it properly defers to cmake to do the build which has dependency resolution implications.

Reply:
You should be able to use `$PWD` to replicate using `--inplace` without the issues.